pyccel/ast/builtin_methods/list_methods.py

```diff
@@ -83,89 +85,169 @@ def append_argument(self):
         """
         return self._append_arg
 
-#==============================================================================
-class ListPop(PyccelInternalFunction) :
+#=======================================================================================
+
+class ListClear(PyccelInternalFunction) :
     """
```

Comment:
Please can you put these classes back in the order that they appear in the devel branch. By reordering everything is showing up as changed. But I am not sure that you have actually changed the implementation of either class?